src/default_action.jl

```diff
@@ -3,9 +3,17 @@ struct ExceptionRethrow end
 default_action(::ExceptionRethrow, mdp, s, ex) = rethrow(ex)
 
 function default_action(f::Function, mdp, s, ex)
-    a = f(s, ex)
-    warn_default(ex, a)
-    return a
+    try
+        a = f(mdp,s,ex)
+        warn_default(ex,a)
+        return a
+    catch e
+        Base.depwarn("""Modify the function definition to take three arguments, i.e., f(mdp,s,ex) instead of f(s,ex). The older version will be deprecated soon.""",
+                :MCTS)
```

Comment:
```suggestion
        Base.depwarn("""Modify the function used for the default action in MCTS to take three arguments, i.e., f(mdp,s,ex) instead of f(s,ex). The older version will be deprecated soon.""",
                :MCTS_default_action_deprecation)
```

Reply:
Also, it is better for warnings to describe what the problem is before saying what the user should do to resolve it. For example
```
The two-argument version of the default action function in MCTS is deprecated. Modify it to take three arguments, i.e., f(mdp,s,ex) instead of f(s,ex).
```